public/config.json

```diff
@@ -9,14 +9,6 @@
       "type": "playlist",
       "contentId": "JSKF03bk"
     },
-    {
-      "enableText": true,
```

Comment:
Can we keep these in the default example config (we can add playlists to the Blender property)